### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by rociac**
Hi, looking forward for your feedback! This is the basic user interface with the mockup logic, it only loops 5 times for testing purposes. Game logic will be on the next pull request.

### CODE REVIEW

**Comment by rociac** [comment]
We are not sure on what to do about those linter errors except for the last one, we just changed it beacause the linter said that we only needed 10 lines in each method. Looking forward for your feedback!

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# tic-tac-toe
\ No newline at end of file
+# Tic-Tac-Toe
+
+Install instructions:
++ install colorize: gem install colorize (mac & linux)
```
Comment (maelfosso) [inline README.md:4]: Good :+1: 

Could you improve your README? The author's name, the description of the project or the link to.

File: bin/main.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+game = Board.new
+
+puts 'Welcome to Tic-Tac-Toe game'
+puts 'Press Enter to continue'
+gets
+
+puts 'Enter name for player 1'
+name = gets.chomp
+
+until weapon == 'X' || weapon == 'O'
```
Comment (maelfosso) [inline bin/main.rb:18]: As linter suggests, you should use an array, write it like `['X', 'O'].include?(weapon)`

File: bin/main.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  puts 'Choose your weapon (X or O)'
+  weapon = gets.chomp.upcase!
```
Comment (maelfosso) [inline bin/main.rb:20]: Please, modify that line. It makes your loop endless. Read first, then, upcase it.

File: lib/board.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  def initialize
+    @possible_wins = [
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 4, 8],
+      [2, 4, 6]]
```
Comment (maelfosso) [inline lib/board.rb:17]: You should put the last closing bracket at the next line, not at the same line than the last element.

File: bin/main.rb [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+require_relative '../lib/player.rb'
+require 'colorize'
+
+weapon = ''
+input = false
+game = Board.new
+
+puts 'Welcome to Tic-Tac-Toe game'
+puts 'Press Enter to continue'
+gets
```
Comment (maelfosso) [inline bin/main.rb:13]: **[OPTIONNAL]** Why?

File: bin/main.rb [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    player_choice = gets.chomp.to_i
+    if game.board[player_choice].is_a? Integer
+      game.board[player_choice] = player2.weapon.blue
+      input = true
+    end
+  end
+
+  game.display_board
+  input = false
+end
```
Comment (maelfosso) [inline bin/main.rb:58]: I suggest you find a way to do that in one step.
Here you're reading player1's input then player2's input. 
Think for a way to read input of the current player, then switch the player and so on.

File: bin/main.rb [hunk 7]
```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require_relative '../lib/board.rb'
+require_relative '../lib/player.rb'
```
Comment (maelfosso) [inline bin/main.rb:4]: Nice :+1: usage of `require_relative`

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello coders,

You did a really nice job. I suggest you do the changes in comments and correct the linters errors.
 Once done, please submit a new code review request.

Don't hesitate to write on Slack, if you have any problems.

**Pull request closed without merging**